api/logout.ts

```diff
@@ -0,0 +1,13 @@
+import {account} from "./appwrite"
+
+const logout = async () =>  {
+    try {
+        const logoutSession = await account.deleteSession('current');
```

Comment:
We don't need a const here. If it fails, the catch will capture the failure. If it succeeds, the code will move onto the next line because of the await.